app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -1,6 +1,28 @@
 <ng-form name="persistentVolumeClaimForm">
   <fieldset ng-disabled="claimDisabled">
-
+    <!--storage class-->
+    <label class="required">Storage Classes</label>
+    <div>
+      <span id="claim-storage-class-help" class="help-block">Storage classes are set by the administrator to define types of storage the users can select.
```

Comment:
@erinboyd @jwforres Any suggestion on this message? As written, I'm not sure how helpful it is to end users.